Delete-update-functionality (#3)
OksanaPR/ContactListAPI: README.md

- Exposes RESTful API with JSON output
- Allows creating a Contact
### generateData.py file.
Usage: `python generateData.py NAME PHONENUMBER EMAIL`
Returns:

- Allows reading the full contact list
### get_all_contacts.py
Usage: `python get_all_contacts.py`
Returns: An array of objects containing all the Contacts.

### get_contact_by_id.py
Usage: `python get_contact_by_id.py ID`
Returns: The contact matching ID, or a message saying it could not be found.

> - Allows deleting a Contact

Reviewer: ```suggestion
```

### delete_contact_by_id.py
Usage: `python delete_contact_by_id.py ID`
Returns: does not return anything

> ### delete_contact_by_id.py

Reviewer: ```suggestion
### delete_contacts.py
```

> Usage: `python deleteContacts.py`

Reviewer: ```suggestion
Usage: `python delete_contacts.py`
```

1. First, run `python main.py` . This will create a new file `pythonsqlite.db` which will contain the ContactsAPI database.

> 2. You can then run any of the following commands

Reviewer: ```suggestion
2. You can then run any of the commands described above.
```

We might also want to move the command explanation below, as it was before? lets discuss this
